ballista/scheduler/src/scheduler_server/mod.rs

```diff
@@ -64,6 +64,7 @@ pub struct SchedulerServer<T: 'static + AsLogicalPlan, U: 'static + AsExecutionP
     pub start_time: u128,
     pub state: Arc<SchedulerState<T, U>>,
     pub(crate) query_stage_event_loop: EventLoop<QueryStageSchedulerEvent>,
+    #[allow(dead_code)] // used by rest-api feature
```

Comment:
Are the remaining uses of `#[allow(dead_code)]` still necessary?

Reply:
they are not, my bad, should be fixed now